runtime/oti/bytecodewalk.h

```diff
@@ -102,6 +102,9 @@ base types: (in the 19bit class index field)
 #define BCV_TAG_BASE_ARRAY_OR_NULL			2		/* set bit means base type array */
 #define BCV_SPECIAL_INIT					4		/* set bit means special init object ("this" for <init>) */
 #define BCV_SPECIAL_NEW						8		/* set bit means special new object (PC offset in upper 28 bits) */
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define BCV_PRIMITIVE_CLASS					16		/* set bit means primitive class */
```

Comment:
The flag is `J9AccPrimitiveValueType`, so I suggest to use `BCV_PRIMITIVE_VALUETYPE`.